mailpoet/assets/js/src/wizard/steps/pitch_mss_step.tsx

```diff
@@ -0,0 +1,35 @@
+import { useRouteMatch, Switch, Route } from 'react-router-dom';
+import { MSSStepFirstPart } from './pitch_mss_step/first_part';
+import { MSSStepSecondPart } from './pitch_mss_step/second_part';
+import { MSSStepThirdPart } from './pitch_mss_step/third_part';
+
+type WelcomeWizardPitchMSSStepPropType = {
+  subscribersCount: number;
+  finishWizard: (redirect_url?: string) => void;
+};
+
+function WelcomeWizardPitchMSSStep({
+  subscribersCount,
+  finishWizard,
+}: WelcomeWizardPitchMSSStepPropType): JSX.Element {
+  const { path } = useRouteMatch();
+
+  return (
+    <Switch>
```

Comment:
I ended up placing the nested routes here. It was the easy place to add it, but I'm not sure if I should have placed everything on wizard.tsx where the parent routes are located.

Reply:
I would do the same since the sub-routes belong to this component.